src/lib/db.js

```diff
@@ -26,22 +27,35 @@ const TABLE_NAMES = [
   'L_LU_TRANSFER_STATUS'
 ]
 
+const MAX_RETRY = 1000
```

Comment:
right, and maybe you would want the number of retries be higher for fulfillments than for initiating new transactions. I'll continue working on this next week.